Makefile

```diff
@@ -8,6 +8,7 @@ mypy:
 	mypy --ignore-missing-imports $(MODULES)
 
 test: lint mypy
+	tests/fixtures/populate.py --s3-bucket $(DSS_S3_BUCKET_TEST_FIXTURES) --gs-bucket $(DSS_GS_BUCKET_TEST_FIXTURES)
```

Comment:
Since we last spoke, I realized that running populate.py in tests is a bad idea since it results in tests racing. In fact I noticed that happen between two branches. So, let's try to figure out something different. Also, if your tests use shared resources like S3 buckets, please be aware of race conditions. In my tests I get around those by randomly generating object key UUIDs.